packages/cf-component-icon/example/basic/component.js

```diff
@@ -3,8 +3,15 @@ import Icon from 'cf-component-icon';
 
 const IconComponent = () => (
   <div>
-    <p>Types:</p>
-    <p>
+    <p>Types (hover for type prop value):</p>
+    <div
+      style={{
+        display: 'grid',
```

Comment:
This was just in the example so I figured it would be ok. I feel like we don't need to support as many browsers in there.